.gitchangelog.rc

```diff
@@ -0,0 +1,57 @@
+output_engine = mustache("markdown")
+
+section_regexps = [
+    ('New', [
+        r'^NEW(:*) \s*?([^\n]*)$',
+     ]),
+    ('Changes', [
+        r'^CHANGE(D*)(:*) \s*?([^\n]*)$',
+     ]),
+    ('Fixes', [
+        r'^FIX(:*) \s*?([^\n]*)$',
+     ]),
+    ('Optimizations', [
+        r'^OPT(:*) \s*?([^\n]*)$',
+     ]),
+    ('Other', None ## Match all lines
+     ),
+
+]
+
+
+ignore_regexps = [
+    r'^Merge*',
+    r'^Automatic merge*',
```

Comment:
is it regular expression or wildcard?
for regexp it should be `.*` instead of just `*`

Reply:
Well, changed a bit to actually be more strict. But tested and working properly.